dlgr/griduniverse/experiment.py

```diff
@@ -1528,13 +1613,14 @@ def send_state_thread(self):
     def game_loop(self):
         """Update the world state."""
         gevent.sleep(0.1)
-        if not self.config.get('replay', False):
+        if not self.config.get("replay", False):
             self.grid.build_labyrinth()
-            logger.info('Spawning food')
-            for i in range(self.grid.num_food):
-                if (i % 250) == 0:
-                    gevent.sleep(0.00001)
-                self.grid.spawn_food()
+            logger.info("Spawning items")
+            for item_type in self.item_config.values():
+                for i in range(item_type["item_count"]):
+                    if (i % 250) == 0:
```

Comment:
I wonder if we need to adjust this sleep check to use an overall loop counter to adjust for the fact that any given item type is likely to have a lower count. Probably doesn't matter for now, but doing the check per item type seems odd to me.